packages/nano.rb

```diff
@@ -24,6 +24,7 @@ class Nano < Package
   })
 
   depends_on 'xdg_base'
+  depends_on 'filecmd'
```

Comment:
This is already a core package so not sure why it was removed.

Reply:
Weird, it wasn't installed on my machine after a fresh install. Oh I know why, the install script is broken right now.